src/main/java/io/zeebe/monitor/zeebe/hazelcast/ZeebeHazelcastService.java

```diff
@@ -10,8 +10,10 @@
 import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
 import org.springframework.stereotype.Component;
 
+@ConditionalOnProperty(name = "zeebe-importer", havingValue = "hazelcast", matchIfMissing = true)
```

Comment:
that smart trick `matchIfMissing` should be explained in the README as well (from the perspective of the user I mean, this is very relevant)

Reply:
Yes, of course. I will mention this in the readme, and also add a description of the use of kafka